.github/workflows/code-checks.yml

```diff
@@ -36,6 +36,8 @@ jobs:
 
     - name: Run pre-commit
       uses: pre-commit/action@v2.0.3
+      with:
+        extra_args: --verbose --all-files
```

Comment:
`verbose` just shows the timings for each hook. `--all-files` is the default `extra_args`